src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -50,13 +51,12 @@
 )
 from llmcompressor.transformers.utils.helpers import detect_last_checkpoint
 
-_LOGGER: logging.Logger = logging.getLogger(__name__)
-
 
 def train(**kwargs):
     """
     CLI entrypoint for running training
     """
+    configure_logger()
```

Comment:
Should we put these configure_logger() calls somewhere higher up in the architecture if we are using this globally for llmcompressor so we need to call it in less places?

Reply:
Since we don't have a single global entry points I think we need to call it in each of train/oneshot/apply

Reply:
Moved this to main()

Reply:
These are no longer needed and are a remnant from trying to get the logger to play nicely as a library. Unfortunately, due to the inability to consistently detect if we are imported as a library for use in other libraries or as a library for a script, this functionally was removed and configure_logger is now the default for any pathways.